src/analytics/parameter.ts

```diff
@@ -470,7 +470,7 @@ export function createStackParameters(scope: Construct): {
   });
 
   const processingFilesLimitParam = new CfnParameter(scope, 'ProcessingFilesLimit', {
-    description: 'Maximum number of files currently being loaded, recommended to be twice the `MaxFilesLimit`.',
+    description: 'Maximum number of files currently being loaded, recommended to be twice the `MaxFilesLimit`.(Deprecated)',
```

Comment:
@tyyzqmf how about the web console supporting removing a parameter in an existing template since next minor version, aka. `1.1.0`?

Reply:
@tyyzqmf let's check if we need remove it from the backend's model.

Reply:
removed **processingFilesLimit** in backend.

Reply:
@amliuyong   Let's also remove the parameter in template.

Reply:
Keep it for the compability for updating existing 1.0.0 pipeline.